easybuild/easyconfigs/i/IPython/IPython-5.7.0-foss-2018a-Python-2.7.14.eb

```diff
@@ -16,23 +16,25 @@ toolchain = {'name': 'foss', 'version': '2018a'}
 
 dependencies = [
     ('Python', '2.7.14'),
-    ('PyZMQ', '17.0.0', '%s-zmq4' % versionsuffix),
+    ('ZeroMQ', '4.2.5'),
+    ('matplotlib', '2.1.2', versionsuffix),
 ]
 
 exts_defaultclass = 'PythonPackage'
+exts_filter = ("python -c 'import %(ext_name)s'", '')
 exts_download_dep_fail = True
 
 exts_list = [
-    ('nose', '1.3.7', {
-        'source_urls': ['https://pypi.python.org/packages/source/n/nose/'],
-        'checksums': ['f1bffef9cbc82628f6e7d7b40d7e255aefaa1adb6a1b1d26c69a8b79e6208a98'],
+    ('pyzmq', '17.0.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pyzmq/'],
+        'checksums': ['0145ae59139b41f65e047a3a9ed11bbc36e37d5e96c64382fcdff911c4d8c3f0'],
+        'modulename': 'zmq',
     }),
     ('nbformat', '4.4.0', {
         'source_urls': ['https://pypi.python.org/packages/source/n/nbformat/'],
         'checksums': ['f7494ef0df60766b7cabe0a3651556345a963b74dbc16bc7c18479041170d402'],
     }),
     ('Pygments', '2.2.0', {
-        'modulename': 'pygments',
```

Comment:
question: why did you remove the modulename here? I thought it was needed for the sanity check `python -c 'import pygments'`, but apparently that is not the case since you installed successfully. Does EB automatically convert to lowercase?

Reply:
Indeed, the default module name is the software name but lowercase (which is the most common pattern for Python packages), so `Pygments` -> `import pygments` is done automatically.